### DISCUSSION TITLE
Downloading models - how to do it?

### DISCUSSION THREAD

**Discussion by pvanallen** [kspviswa/PyOMlx#4] [Q&A]
I don't see documentation on how to download a model for use with PyOMlx and pyollmx. What is the process?

**Comment by kspviswa** [comment]
PyOMlx is a model server for Apple MlX models. You can download MlX models directly from huggingface here https://huggingface.co/mlx-community .

PyOllaMx is a chat interact utility that can use both Ollama and MlX models (via PyOMlx) . So for ollama models, simply use ollama to download models . You can look here https://ollama.com/library .

I have a roadmap item to streamline and simply this process by adding mode discovery and download functionality directly within PyOllaMx app itself. It's coming soon 😊
